docs/example_plot_time_series.ipynb

```diff
@@ -0,0 +1,1458 @@
a4g/6ws7h5m4JYki4eXJCnr4K8OofvA8g/1x7cfyBpYSRUbdVDI9W1ksVgu3wSAcozQDQAAAFzhju9YJ3cvb4drvgOqRuvPVf9R2sZvVKlZZxXk5ejQms8kSXlZ6Wa/iNZ3KqBqtNw8rLLv264D332ogrxshTbtfLmnAZRLhG4AAADgCnd8+1oFR8fK7f8fyZYkn4pVVL3rIP258j86uHa+LG5uqtTkFnn4BqroQ4oiYrubX/uGVVNBXo7SNi0mdAOXCaEbAAAAuIJl/rlbOScOK+S2R4q1BUfHKjg6VnlZGXLztEqy6MjmJbIGVTrneH4RNZX6/VcqOJsnNw9PJ1YOQCJ0AwAAAFe0Y9vWyDesunxDq56zj6dfoNnXzd1TAdXqn7Pv6aP75e7tR+AGLhNCNwAAAOAC+bnZyklPM5dzMo7q9JE/5OHtLy9byF99cs4offdGVW5/f4ljHPlpmfwr15Gbp7dO/bFdf66ep8pt75aHt5+kv+54fjYrQ34RtWXx8NSpP7Yr7fuFCm3R1fkTBCCJ0A0AAAC4xOnUFO357AVz+eCq/0iSguvfZN6l/OSu72VICo5udY4x9urwhi9VkJcj7+AIVb2ln0LqtzbbLW7uOpq8XH+u/I8kQ9agMFW++QFVbNjOafMC4IjQDQAAALhAQNVoNX18znn7VGx0syo2uvmc7dVv/b/zbh9Yo6ECazQsVX0AyobbhbsAAAAAAIDSIHQDAAAAAOAkhG4AAAAAAJyE0A0AAAAAgJMQugEAAAAAcBJCNwAAAAAATkLoBgAAAADASQjdAAAAAAA4CaEbAAAAAAAncWnoXrNmjW6//XZFRkbKYrFowYIFDu2GYWjChAmKiIiQj4+POnXqpD179jj0OXHihHr16iWbzaagoCANGDBAmZmZDn22bt2qNm3ayNvbW1FRUZo2bZqzpwYAAAAAgGtDd1ZWlho1aqQ33nijxPZp06bptdde06xZs/TDDz/Iz89PcXFxys7ONvv06tVLO3bs0LJly7Ro0SKtWbNGDz/8sNlut9vVuXNnVatWTZs3b9b06dM1adIkvfPOO06fHwAAAACgfPNw5c67du2qrl27lthmGIZeeeUVjR8/Xt27d5ckffjhhwoLC9OCBQt03333aefOnVqyZIk2bdqk5s2bS5Jef/113XrrrXrxxRcVGRmpuXPnKjc3Vx988IG8vLxUv359JScn6+WXX3YI5wAAAAAAlLUr9prulJQUpaamqlOnTua6wMBAtWzZUklJSZKkpKQkBQUFmYFbkjp16iQ3Nzf98MMPZp+2bdvKy8vL7BMXF6fdu3fr5MmTJe47JydHdrvd4QUAAAAAwKW6YkN3amqqJCksLMxhfVhYmNmWmpqq0NBQh3YPDw8FBwc79ClpjKL7+LupU6cqMDDQfEVFRf3zCQEAAAAAyp0rNnS70rhx45SRkWG+Dhw44OqSAAAAAABXoSs2dIeHh0uS0tLSHNanpaWZbeHh4Tpy5IhD+9mzZ3XixAmHPiWNUXQff2e1WmWz2RxeAAAAAABcqis2dNeoUUPh4eFavny5uc5ut+uHH35QbGysJCk2Nlbp6enavHmz2WfFihUqKChQy5YtzT5r1qxRXl6e2WfZsmWqW7euKlSocJlmAwAAAAAoj1waujMzM5WcnKzk5GRJf908LTk5Wfv375fFYtGIESP03HPP6euvv9a2bdvUp08fRUZGqkePHpKk6OhodenSRYMGDdLGjRu1fv16DRs2TPfdd58iIyMlSQ888IC8vLw0YMAA7dixQ/PmzdOrr76q+Ph4F80aAAAAAFBeuPSRYT/++KNuvvlmc7kwCPft21cJCQkaM2aMsrKy9PDDDys9PV033XSTlixZIm9vb3ObuXPnatiwYerYsaPc3NzUs2dPvfbaa2Z7YGCgvv32Ww0dOlTNmjVTxYoVNWHCBB4XBgAAAABwOothGIari7jS2e12BQYGKiMj46q7vrvZ6A9dXQIA4Dw2T+/j6hLKBT4PAeDKd7V9Jl5sTrxir+kGAAAAAOBqR+gGAAAAAMBJCN0AAAAAADgJoRsAAAAAACchdAMAAAAA4CSEbgAAAAAAnITQDQAAAACAkxC6AQAAAABwEkI3AAAAAABOQugGAAAAAMBJCN0AAAAAADgJoRsAAAAAACchdAMAAAAA4CSEbgAAAAAAnITQDQAAAACAkxC6AQAAAABwEkI3AAAAAABOQugGAAAAAMBJCN0AAAAAADgJoRsAAAAAACchdAMAAAAA4CSEbgAAAAAAnITQDQAAAACAkxC6AQAAAABwEkI3AAAAAABOQugGAAAAAMBJCN0AAAAAADgJoRsAAAAAACchdAMAAAAA4CSEbgAAAAAAnKRche433nhD1atXl7e3t1q2bKmNGze6uiQAAAAAwDWs3ITuefPmKT4+XhMnTtRPP/2kRo0aKS4uTkeOHHF1aQAAAACAa1S5Cd0vv/yyBg0apIceekgxMTGaNWuWfH199cEHH7i6NAAAAADANcrD1QVcDrm5udq8ebPGjRtnrnNzc1OnTp2UlJRUrH9OTo5ycnLM5YyMDEmS3W53frFlLD/njKtLAACcx9X42XI14vMQAK58V9tnYmG9hmGct1+5CN3Hjh1Tfn6+wsLCHNaHhYVp165dxfpPnTpVkydPLrY+KirKaTUCAMqnwNcHu7oEAACuCFfrZ+KpU6cUGBh4zvZyEbov1bhx4xQfH28uFxQU6MSJEwoJCZHFYnFhZUD5ZrfbFRUVpQMHDshms7m6HAAAXIbPRMD1DMPQqVOnFBkZed5+5SJ0V6xYUe7u7kpLS3NYn5aWpvDw8GL9rVarrFarw7qgoCBnlgjgEthsNv6DAQCA+EwEXO18R7gLlYsbqXl5ealZs2Zavny5ua6goEDLly9XbGysCysDAAAAAFzLysWRbkmKj49X37591bx5c91www165ZVXlJWVpYceesjVpQEAAAAArlHlJnTfe++9Onr0qCZMm
```

Comment:
I'm just leaving this comment here (I also mentioned in my PR https://github.com/UBC-MDS/PyXplor/pull/87) but let's meet and figure out how to consolidate our sections to what @phchen5 has already pushed to main!